.github/workflows/deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+
+  publish-doc:
+    runs-on: ubuntu-20.04
+    permissions:
+      pages: write
+      id-token: write
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
@ranjanrak can't we update node-version to 20 ? is there any specific reason keeping node 16 ?